ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -600,20 +612,23 @@ protected ValidationResult<RpcErrorType> validateBlobs(
     return ValidationResult.valid();
   }
 
-  private boolean validateExcessBlobGas(
+  private Optional<BlobGas> validateExcessBlobGas(
       final BlockHeader header, final BlockHeader parentHeader, final ProtocolSpec protocolSpec) {
-    BlobGas calculatedBlobGas =
+    BlobGas calculated =
         ExcessBlobGasCalculator.calculateExcessBlobGasForParent(protocolSpec, parentHeader);
-    return header.getExcessBlobGas().orElse(BlobGas.ZERO).equals(calculatedBlobGas);
+    BlobGas actual = header.getExcessBlobGas().orElse(BlobGas.ZERO);
+
+    return calculated.equals(actual) ? Optional.empty() : Optional.of(calculated);
   }
 
-  private boolean validateBlobGasUsed(
+  private Optional<Long> validateBlobGasUsed(
```

Comment:
unit tests also a comment to explain the value that is returned.

Reply:
Added tests and comments